### ISSUE TITLE
should-fix

### ISSUE THREAD

**Issue by kimchristian50** [jdethington/wdd430-portfolio#3]
https://github.com/jdethington/wdd430-portfolio/blob/95da7246d6abd46883ec8b3cbe128791eb077731/components/ProjectCard.tsx#L25

should-fix: Add screen reader notice for external links

In ProjectCard.tsx, the link opens in a new tab (target="_blank"). Adding aria-label="${title} (opens in a new tab)" or an accessible indicator ensures screen reader users are aware the link will open outside the current site.

I was digging deep for this should-fix. :) I'm sure my site from last week lacks the aria-labels.
